instrumentation/action_view/Gemfile

```diff
@@ -11,5 +11,7 @@ gemspec
 group :test do
   gem 'opentelemetry-instrumentation-active_support', path: '../active_support'
   gem 'opentelemetry-instrumentation-base', path: '../base'
+  gem 'nokogiri', force_ruby_platform: true
```

Comment:
can you try:

```ruby
  gem 'nokogiri', platforms: [:ruby, :x64_mingw]
```

This should at least work on Windows Ruby 3.0 and earlier, and if you are installing a relatively recent Bundler, it will work on Windows Ruby 3.1 as well.